creator-node/src/middlewares/readOnly/readOnlyMiddleware.js

```diff
@@ -1,15 +1,19 @@
 const { sendResponse, errorResponseServerError } = require('../../apiHelpers')
 const config = require('../../config')
 
-const exclusionsRegex = new RegExp(/(vector_clock)/)
 /**
  * Middleware to block all non-GET api calls if the server should be in "read-only" mode
  */
 function readOnlyMiddleware(req, res, next) {
   const isReadOnlyMode = config.get('isReadOnlyMode')
+  console.log("config.get('spID')", config.get('spID'))
+  const spIDNotDefined = config.get('spID') === 0 // true if not a valid spID
```

Comment:
with convict constraints can this ever be misconfigured to be <1 or empty to be '' or undefined? if so maybe it should be `const spIDNotDefined = !config.get('spID') || config.get('spID') <= 0`?

Reply:
it's supposed to be a number so it should never be a string or undefined but i like your condition so just added